test/dotnet-new.Tests/GivenThatIWantANewAppWithSpecifiedType.cs

```diff
@@ -34,6 +34,13 @@ public class GivenThatIWantANewAppWithSpecifiedType : TestBase
             string projectType,
             bool useNuGetConfigForAspNet)
         {
+            if (language == "F#" && !EnvironmentInfo.HasSharedFramework("netcoreapp1.0"))
```

Comment:
I need to turn these tests on again, though they won't run anywhere, since no CLI carries 1.0 anymore.